packages/headless/src/api/commerce/commerce-api-params.ts

```diff
@@ -57,7 +57,7 @@ export type UserParams = (UserIdRequired | EmailRequired | UserIdAndEmail) & {
 };
 
 export interface CartItemParam {
-  productId: string;
+  sku: string;
```

Comment:
Would naming this `variantId` cause more confusion? `variantId` seems more generic than `sku`.
```suggestion
  variantId: string;
```

Reply:
That's a good question. I think I saw more often `sku`, but I might be tainted with the commerce ecosystem?

Reply:
I'd tend to use `sku` because if I'm not mistaken, that's the field name in the default catalog configuration, and this is the field we use in our documentation, e.g.: https://docs.coveo.com/en/m53g0506/coveo-for-commerce/catalog-variant-data-and-product-groupings